packages/react-button/webpack.serve.config.js

```diff
@@ -1,4 +0,0 @@
-const resources = require('@fluentui/scripts/webpack/webpack-resources');
-
-// react-button has a legacy demo app to help test the IE 11 polyfill (storybook doesn't work in IE).
-module.exports = resources.createLegacyDemoAppConfig();
```

Comment:
what is the legacy demo app it speaks of ?

Reply:
This allows a build app without storybook that will work in IE11, an itself was located in `packages/react-examples/src/react-button/demo/index.tsx`.

Reply:
ah right, thx for the explanation